README.md

```diff
@@ -1,26 +1,47 @@
 
 # python.cz
 
-Czech Python community homepage.
+Czech Python community homepage
 
 [![Build Status](https://travis-ci.org/pyvec/python.cz.svg?branch=master)](https://travis-ci.org/pyvec/python.cz)
 [![Test Coverage](https://coveralls.io/repos/github/pyvec/python.cz/badge.svg?branch=master)](https://coveralls.io/github/pyvec/python.cz?branch=master)
 
-## Installation
+### Requirements
 
-The code is **Python 3** (the production Python version is 3.4).
+-  The code is **Python 3** (the production Python version is 3.4)
+-  The site uses GitHub API. For certian pages to work correctly, you need to set the `GITHUB_TOKEN` environment variable to a [GitHub Personal Access Token](https://github.com/settings/tokens) (no scopes needed)
+
+### Get Started
+
+#### Development
+
+Installation:
 
 ```sh
-$ pip install .
+$ pip install -e .[tests]
+```
+
+Development server:
+
+```sh
+$ export GITHUB_TOKEN=...
+$ pythoncz run
 ```
 
-### Development
+#### Production
+
+Installation:
+
+```sh
+$ pip install .
+$ pip install gunicorn
+```
 
-The site uses GitHub API. For certian pages to work correctly, you need to set the `GITHUB_TOKEN` environment variable to a [GitHub Personal Access Token](https://github.com/settings/tokens) (no scopes needed).
+Server:
 
 ```sh
 $ export GITHUB_TOKEN=...
-$ python runserver.py
+$ gunicorn pythoncz.app:app
```

Comment:
Sice moje mašina není produkce, ale kdyby byla, tak mi to nefunguje:

```console
$ gunicorn pythoncz.app:app
[2017-09-12 21:54:03 +0200] [32578] [INFO] Starting gunicorn 19.7.1
[2017-09-12 21:54:03 +0200] [32578] [INFO] Listening at: http://127.0.0.1:8000 (32578)
[2017-09-12 21:54:03 +0200] [32578] [INFO] Using worker: sync
[2017-09-12 21:54:03 +0200] [32581] [INFO] Booting worker with pid: 32581
```
```console
$ http http://127.0.0.1:8000
HTTP/1.1 404 NOT FOUND
Connection: close
Content-Length: 4183
Content-Type: text/html; charset=utf-8
Date: Tue, 12 Sep 2017 19:56:00 GMT
Server: gunicorn/19.7.1

... (naše HTML pro 404 stránku) ...
```
(A to se děje i když hodím python.cz na localhost v `/etc/hosts`.)

Reply:
Když to u sebe spustíš takhle, tak to ale funguje (na http://localhost:8000/):

```shell
$ SERVER_NAME=localhost:8000 gunicorn pythoncz.app:app
```

Reply:
Máš pravdu.

Reply:
Navrhuji přidat větu:

> Produkční nasazení na lokálním počítači nebude fungovat. Pokud ho chcete jen vyzkoušet, nastavte proměnnou prostředí `SERVER_NAME` na `localhost:8000` a přistupte na [localhost:8000](http://localhost:8000/).

Reply:
Přidal jsem v 46e4c5c